### ISSUE TITLE
NEW FEATURE : Desktop = wmderland

### ISSUE THREAD

**Issue by arcolinuxz** [dylanaraps/neofetch#1935]
We are developing a new desktop called "wmderland".

I noticed the name is not that nice in the current neofetch.

Maybe we should add some code to set the WM correct.

Thanks in advance for considering it.
Always ready to test and help.

![image](https://user-images.githubusercontent.com/36679406/138322121-84abba1a-cad5-423e-ac7c-5bf0f5acded9.png)

**Comment by MichaelStraube** [comment]
Can you post the output of `xprop -root -notype _NET_SUPPORTING_WM_CHECK` ?

**Comment by arcolinuxz** [comment]
![ArcoLinux-2021-11-14_07-21](https://user-images.githubusercontent.com/36679406/141670116-dc80e568-a35d-4029-84d9-6a356d5e9cb7.jpg)

**Comment by arcolinuxz** [comment]
![image](https://user-images.githubusercontent.com/36679406/141670171-b03a738d-9cc7-4d4e-a455-b1175e7f5670.png)

**Comment by MichaelStraube** [comment]
Thank you. And now, what is the output of:
```bash
id=$(xprop -root -notype _NET_SUPPORTING_WM_CHECK)
id=${id##* }
xprop -id "$id" -notype -len 100 -f _NET_WM_NAME 8t
```

**Comment by MichaelStraube** [comment]
I think we could just add wmderland to this grep https://github.com/dylanaraps/neofetch/blob/e784522206f4f60b37a4186574ba6b3bc3163247/neofetch#L1939

or find out if there is something wrong in this block https://github.com/dylanaraps/neofetch/blob/e784522206f4f60b37a4186574ba6b3bc3163247/neofetch#L1950

**Comment by MichaelStraube** [comment]
> Thank you. And now, what is the output of:
> 
> ```shell
> id=$(xprop -root -notype _NET_SUPPORTING_WM_CHECK)
> id=${id##* }
> xprop -id "$id" -notype -len 100 -f _NET_WM_NAME 8t
> ```

It should look like this:
```bash
_NET_WM_NAME = "wmderland" _NET_SUPPORTING_WM_CHECK: window id # 0x200001
```

My guess is that wmderland does not set _NET_WM_NAME correctly.

**Comment by arcolinuxz** [comment]
![ArcoLinux_2021-11-14_18-33-34](https://user-images.githubusercontent.com/36679406/141691780-7ab42140-f799-403d-b213-90eadd1a8cc1.png)

Is this ok?

**Comment by arcolinuxz** [comment]
![ArcoLinux_2021-11-14_18-35-02](https://user-images.githubusercontent.com/36679406/141691834-a7963c83-cf82-4de7-ad21-979d4dfe398b.png)
These are the environment variables you can use in case wmderland does set it right
if on wmderland then use ...

**Comment by MichaelStraube** [comment]
> ![ArcoLinux_2021-11-14_18-33-34](https://user-images.githubusercontent.com/36679406/141691780-7ab42140-f799-403d-b213-90eadd1a8cc1.png)
> 
> Is this ok?

No, according to https://specifications.freedesktop.org/wm-spec/wm-spec-latest.html#idm46515148903664 it isn't.
Looks like the wmderland Window Manager isn't 100% EWMH compliant.

If this cannot be fixed in wmderland, I would add it to the non-EWMH list.
https://github.com/dylanaraps/neofetch/blob/e784522206f4f60b37a4186574ba6b3bc3163247/neofetch#L1939

Let's wait what @dylanaraps say.

**Comment by dylanaraps** [comment]
> If this cannot be fixed in wmderland, I would add it to the non-EWMH list.

:+1:

**Comment by arcolinuxz** [comment]
I have asked assistance on Wmderland github.

Maybe they can help.

https://github.com/aesophor/wmderland/issues/63
